### PR TITLE
fix: Invalid CODE tags nesting in DL

### PR DESCRIPTION
**Pull request by nschonni**
***
<a href="https://pr-preview.s3.amazonaws.com/nschonni/scroll-to-text-fragment/pull/164.html" title="Last updated on Feb 8, 2021, 4:41 PM UTC (3f63052)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/scroll-to-text-fragment/164/d7a3c5d...nschonni:3f63052.html" title="Last updated on Feb 8, 2021, 4:41 PM UTC (3f63052)">Diff</a>

### CODE REVIEW

**Comment by bokand** [comment]
Thanks for the fix!

This does lose the code formatting which I think is preferable - could we instead move the `<code>` tags (or use the more readable backticks) inside the `<dd>` and `<dt>` elements instead?

**Comment by nschonni** [comment]
Current version:
Current version

PR Version:
PR version

Maybe you can suggest some inline changes, since I'm not sure what parts should actually be code spans in this

File: index.bs [hunk 1]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
-    <dt>
-      <dfn>ExplicitChar</dfn> ::=
-    </dt>
-    <dd>
-      [a-zA-Z0-9] | "!" | "$" | "'" | "(" | ")" | "*" | "+" | "." | "/" | ":" |
-      ";" | "=" | "?" | "@" | "_" | "~" | "&" | "," | "-"
-    </dd>
-  </code>
+  <dt>
+    <dfn id="fragmentdirectiveproduction">FragmentDirective</dfn> ::=
```
Comment (bokand) [inline index.bs:511]:
```suggestion
    <dfn id="fragmentdirectiveproduction">`FragmentDirective`</dfn> `::=`
```

File: index.bs [hunk 2]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
+  </dt>
+  <dd>
+    ([=TextDirective=] | [=UnknownDirective=]) ("&" [=FragmentDirective=])?
```
Comment (bokand) [inline index.bs:514]:
```suggestion
    <code>([=TextDirective=] | [=UnknownDirective=]) ("&" [=FragmentDirective=])?</code>
```

File: index.bs [hunk 3]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
+  </dd>
+  <dt>
+    <dfn>UnknownDirective</dfn> ::=
```
Comment (bokand) [inline index.bs:517]:
```suggestion
    <dfn>`UnknownDirective`</dfn> `::=`
```

File: index.bs [hunk 4]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
+  </dt>
+  <dd>
+    [=CharacterString=]
```
Comment (bokand) [inline index.bs:520]:
```suggestion
    <code>[=CharacterString=]</code>
```

File: index.bs [hunk 5]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
+  </dd>
+  <dt>
+    <dfn>CharacterString</dfn> ::=
```
Comment (bokand) [inline index.bs:523]:
```suggestion
    <dfn>`CharacterString`</dfn> `::=`
```

File: index.bs [hunk 6]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
+  </dt>
+  <dd>
+    ([=ExplicitChar=] | [=PercentEncodedChar=])+
```
Comment (bokand) [inline index.bs:526]:
```suggestion
    <code>([=ExplicitChar=] | [=PercentEncodedChar=])+</code>
```

File: index.bs [hunk 7]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
+  </dd>
+  <dt>
+    <dfn>ExplicitChar</dfn> ::=
```
Comment (bokand) [inline index.bs:529]:
```suggestion
    <dfn>`ExplicitChar`</dfn> `::=`
```

File: index.bs [hunk 8]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
+  </dt>
+  <dd>
+    [a-zA-Z0-9] | "!" | "$" | "'" | "(" | ")" | "*" | "+" | "." | "/" | ":" |
```
Comment (bokand) [inline index.bs:532]:
```suggestion
    <code>[a-zA-Z0-9] | "!" | "$" | "'" | "(" | ")" | "*" | "+" | "." | "/" | ":" |
```

File: index.bs [hunk 9]
```diff
@@ -507,42 +507,34 @@ directive input|, run these steps:
...
+    ";" | "=" | "?" | "@" | "_" | "~" | "&" | "," | "-"
```
Comment (bokand) [inline index.bs:533]:
```suggestion
    ";" | "=" | "?" | "@" | "_" | "~" | "&" | "," | "-"</code>
```

File: index.bs [hunk 10]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
     explicitly used in the [=TextDirective=] syntax, that is "&", "-", and ",",
     which must be percent-encoded.
   </div>
-
-  <code>
-    <dt><dfn>PercentEncodedChar</dfn> ::=</dt>
-    <dd>"%" [a-zA-Z0-9]+</dd>
-  </code>
+  </dd>
+  <dt><dfn>PercentEncodedChar</dfn> ::=</dt>
```
Comment (bokand) [inline index.bs:575]:
```suggestion
  <dt><dfn>`PercentEncodedChar`</dfn> `::=`</dt>
```

File: index.bs [hunk 11]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dd>"%" [a-zA-Z0-9]+</dd>
```
Comment (bokand) [inline index.bs:576]:
```suggestion
  <dd><code>"%" [a-zA-Z0-9]+</code></dd>
```

File: index.bs [hunk 12]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
-
-  <code>
-    <dt><dfn>TextDirectiveExplicitChar</dfn> ::=</dt>
-    <dd>
-      [a-zA-Z0-9] | "!" | "$" | "'" | "(" | ")" | "*" | "+" | "." | "/" | ":" |
-      ";" | "=" | "?" | "@" | "_" | "~"
-    </dd>
-  </code>
-
+  <dt><dfn>TextDirective</dfn> ::=</dt>
```
Comment (bokand) [inline index.bs:552]:
```suggestion
  <dt><dfn>`TextDirective`</dfn> `::=`</dt>
```

File: index.bs [hunk 13]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dd>"text=" [=TextDirectiveParameters=]</dd>
```
Comment (bokand) [inline index.bs:553]:
```suggestion
  <dd><code>"text=" [=TextDirectiveParameters=]</code></dd>
```

File: index.bs [hunk 14]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dt><dfn>TextDirectiveParameters</dfn> ::=</dt>
```
Comment (bokand) [inline index.bs:554]:
```suggestion
  <dt><dfn>`TextDirectiveParameters`</dfn> `::=`</dt>
```

File: index.bs [hunk 15]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dd>
+    ([=TextDirectivePrefix=] ",")? [=TextDirectiveString=]
```
Comment (bokand) [inline index.bs:556]:
```suggestion
    <code>
    ([=TextDirectivePrefix=] ",")? [=TextDirectiveString=]
```

File: index.bs [hunk 16]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+    ("," [=TextDirectiveString=])?  ("," [=TextDirectiveSuffix=])?
```
Comment (bokand) [inline index.bs:557]:
```suggestion
    ("," [=TextDirectiveString=])?  ("," [=TextDirectiveSuffix=])?
    </code>
```

File: index.bs [hunk 17]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  </dd>
+  <dt><dfn>TextDirectivePrefix</dfn> ::=</dt>
```
Comment (bokand) [inline index.bs:559]:
```suggestion
  <dt><dfn>`TextDirectivePrefix`</dfn> `::=`</dt>
```

File: index.bs [hunk 18]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dd>[=TextDirectiveString=]"-"</dd>
```
Comment (bokand) [inline index.bs:560]:
```suggestion
  <dd><code>[=TextDirectiveString=]"-"</code></dd>
```

File: index.bs [hunk 19]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dt><dfn>TextDirectiveSuffix</dfn> ::=</dt>
```
Comment (bokand) [inline index.bs:561]:
```suggestion
  <dt><dfn>`TextDirectiveSuffix`</dfn> `::=`</dt>
```

File: index.bs [hunk 20]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dd>"-"[=TextDirectiveString=]</dd>
```
Comment (bokand) [inline index.bs:562]:
```suggestion
  <dd><code>"-"[=TextDirectiveString=]</code></dd>
```

File: index.bs [hunk 21]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dt><dfn>TextDirectiveString</dfn> ::=</dt>
```
Comment (bokand) [inline index.bs:563]:
```suggestion
  <dt><dfn>`TextDirectiveString`</dfn> `::=`</dt>
```

File: index.bs [hunk 22]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dd>([=TextDirectiveExplicitChar=] | [=PercentEncodedChar=])+</dd>
```
Comment (bokand) [inline index.bs:564]:
```suggestion
  <dd><code>([=TextDirectiveExplicitChar=] | [=PercentEncodedChar=])+</code></dd>
```

File: index.bs [hunk 23]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dt><dfn>TextDirectiveExplicitChar</dfn> ::=</dt>
```
Comment (bokand) [inline index.bs:565]:
```suggestion
  <dt><dfn>`TextDirectiveExplicitChar`</dfn> `::=`</dt>
```

File: index.bs [hunk 24]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+  <dd>
```
Comment (bokand) [inline index.bs:566]:
```suggestion
  <dd>
  <code>
```

File: index.bs [hunk 25]
```diff
@@ -557,52 +549,31 @@ The <dfn>text fragment directive</dfn> is one such [=fragment directive=] that
...
+    [a-zA-Z0-9] | "!" | "$" | "'" | "(" | ")" | "*" | "+" | "." | "/" | ":" |
+    ";" | "=" | "?" | "@" | "_" | "~"
```
Comment (bokand) [inline index.bs:568]:
```suggestion
    ";" | "=" | "?" | "@" | "_" | "~"
    </code>
```

**Review by bokand — APPROVED** [review]
Added code/backticks (looks like backticks don't work well with some of the autolinking syntax so code is still necessary)

lgtm otherwise

**Comment by bokand** [comment]
Thanks!

**Pull request merged**